scripts/beforepack.js

```diff
@@ -0,0 +1,14 @@
+// Copyright (c) 2016-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+var fs = require('fs');
+var path = require('path');
+
+exports.default = async function beforePack(context) {
+    // The debian packager (fpm) complains when the directory to output the package to doesn't exist
+    // So we have to manually create it first
+    var dir = path.join(context.outDir, context.packager.appInfo.version)
```

Comment:
Not familiar how the `context` is set but do we need a nil check here for anything in `context.packager.appInfo` ?

Reply:
Not according to the types, no. So if we do get an `undefined` error we probably should be failing hard.